eclipse/tern.eclipse.ide.server.nodejs.core/src/tern/eclipse/ide/server/nodejs/core/NodejsDebuggersManager.java

```diff
@@ -0,0 +1,47 @@
+/**
+ *  Copyright (c) 2015 Genuitec LLC.
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  which accompanies this distribution, and is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *
+ *  Contributors:
+ *  Piotr Tomiak <piotr@genuitec.com> - initial API and implementation
+ */
+package tern.eclipse.ide.server.nodejs.core;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import tern.eclipse.ide.server.nodejs.internal.core.debugging.webclipse.WebclipseDebugger;
+
+public class NodejsDebuggersManager {
+
+	private static Map<String, INodejsDebugger> debuggers = new LinkedHashMap<String, INodejsDebugger>();
+
+	static {
+		// Add supported debuggers
+		addDebugger(new WebclipseDebugger());
```

Comment:
At first, thanks @piotrtomiak for your work. We have each an opinion : pros/cons extension point and before merging your code, I would like just to know if other guys like JBoss guys are OK with your proposition to have debugger list hard coded (with uninstalled label of debugger if it is not installed).

@vrubezhny @dgolovin @maxandersen @pascalleclercq @gamerson @PaulVI @fbricon @piotrtomiak @jabby are you OK to have default hard coded debugger list (like Webclipse) or do you prefer having an extension point to fill debugger combo (it means that we must create a new plugin which uses the extension point to provide a debugger)?